c_plus_plus/src/model/classifier/classifier.cpp

```diff
@@ -0,0 +1,30 @@
+#include <cmath>
+
+#include "classifier.h"
+
+namespace Model
+{
+
+std::vector<int> Classifier::classify(std::vector<SourcePosition> audioPositions, std::vector<SphericalAngleRect> imagePositions, int rangeThreshold)
```

Comment:
We should add a "const ref"  to audioPositions and imagePositions and to rangeThreshold.

Reply:
why?

Reply:
because it is a best practice. The reference to avoid copies and since you use a reference you need a const. Those parameters are just inputs, not outputs. So to tell the person who's using your code that you're not gonna change what he sent you, the const is usefull that way. Slack me if you don't understand.